Tasks/PWGHF/taskJpsi.cxx

```diff
@@ -47,6 +47,10 @@ struct TaskJpsi {
   Configurable<bool> d_modeJpsiToMuMu{"d_modeJpsiToMuMu", false, "Perform Jpsi to mu+mu- analysis"};
   Configurable<double> cutYCandMax{"cutYCandMax", -1., "max. cand. rapidity"};
   Configurable<std::vector<double>> bins{"pTBins", std::vector<double>{hf_cuts_jpsi_toee::pTBins_v}, "pT bin limits"};
+  Configurable<int> noPid{"noPid", 0, "Pid Flag for Jpsi"};
+  Configurable<int> tofPid{"tofPid", 0, "Pid Flag for Jpsi"};
+  Configurable<int> richPid{"richPid", 0, "Pid Flag for Jpsi"};
+  Configurable<int> combPid{"combPid", 0, "Pid Flag for Jpsi"};
```

Comment:
You use the same and wrong description for all parameters.
These are not PID flags. These are selection switches. Why are they `int`?
`noPid` does not switch off PID, it selects with topological cuts. The name is misleading.
The other names are also quite cryptic.

Reply:
Hi Vit, thanks for your comments. you're right, I have committed a new version. please have a look.